src/panel/applets/trash/trash_applet.c

```diff
@@ -17,6 +17,7 @@
 #include "trash_notify.h"
 #include "trash_popover.h"
 #include "trash_settings.h"
+#include <popover.h>
```

Comment:
I think we might want this to be:

```c
#include "popover.h
```

I think by using brackets, it will point to the system version of the header, whereas we probably want to use the current version in the source tree. @JoshStrobl am I correct in this assumption?